lib/git_reflow.rb

```diff
@@ -157,7 +155,11 @@ def deliver(options = {})
             say "There were problems commiting your feature... please check the errors above and try again.", :error
           end
         elsif !existing_pull_request.build_status.nil? and existing_pull_request.build_status != "success"
```

Comment:
I think this is a good time to refactor the `elsif`s here into the `PullReqest` itself and then just have all of these consolidated here into something like:
```ruby
  #...
  else
    say existing_pull_request.rejection_message, :deliver_halted
  end
```

where:

```ruby
```